src/main/scala/utils/MyOneHot.scala

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import chisel3._
+import chisel3.util._
+
+object OHToUIntStartOne {
+  def apply(in: UInt): UInt = {
+    OHToUInt(in << 1)
+  }
+}
+
+/*
+object OHToUIntEndMost {
+  def apply(in: UInt): UInt = {
+    val res = UInt(64.W)
+    when(in.orR){
+      res := OHToUInt(in << 1)
+    }.otherwise{
+      res := 1.U << in.getWidth
+    }
+    res
+  }
+}
+*/
```

Comment:
It is recommended not to write it in a new file. You can add it to BitUtil

Reply:
thanks for reminding. It is been fixed.